test/e2e/tests/book-appointment.spec.ts

```diff
@@ -80,15 +84,23 @@ test.describe('book an appointment', {
     // now fill out the book selection dialog with booking requester's info and book it
     await bookingPage.finishBooking(APPT_BOOKING_REQUESTER_NAME, APPT_BOOKING_REQUESTER_EMAIL);
 
-    // 'boooking request sent' text appears twice, once in the pop-up and once in underlying page
-    await expect(bookingPage.requestSentTitleText.first()).toBeVisible({ timeout: 60_000 });
-    await expect(bookingPage.requestSentTitleText.nth(1)).toBeVisible();
+    if (APPT_TARGET_ENV == 'dev') {
+      // when running against local dev environment after a slot is booked it doesn't require
+      // confirmation by the host user, so the event is automatically booked and the title text
+      // will be different to reflect that the appointment is booked (not pending/requested)
+      await expect(bookingPage.eventBookedTitleText).toBeVisible({ timeout: 60_000 });
+    } else {
+      // in production and stage after a slot is booked it requires confirmation from the host user first
+      // 'boooking request sent' text appears twice, once in the pop-up and once in underlying page
+      await expect(bookingPage.requestSentTitleText.first()).toBeVisible({ timeout: 60_000 });
+      await expect(bookingPage.requestSentTitleText.nth(1)).toBeVisible();
+    }
```

Comment:
Do we want to adjust this? It's just a setting in the schedule settings. We could run a separate test for auto confirmation.

Reply:
Ohhh I forgot about the `booking confirmation` UI setting hah I must have turned that off on my local env at some point. I see it's enabled by default so yeah I'll change this test to always expect it regardless the env, as when running the tests locally they should mirror the settings in stage/prod as best we can. Good catch, thanks!